game-of-life-app/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import { Route, Switch } from "react-router-dom";
+
+// pages
+import Header from "./components/pages/Header";
+import About from "./components/pages/About";
+import Credits from "./components/pages/Credits";
+import { PageNotFound } from "./components/pages/PageNotFound";
+import Game from "./components/game/Game";
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Header title="Conway's Game of Life" />
+        <Switch>
+          <Route exact path="/" render={props => <Game {...props} />} />
+          <Route exact path="/about" component={About} />
+          <Route exact path="/credits" component={Credits} />
+          <Route component={PageNotFound} />
```

Comment:
Good routing here! All the pages that you need for MVP are here, can't wait to see them!